saleor/site/migrations/0035_sitesettings_separate_state_remove_order_settings.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 3.2.16 on 2022-12-14 09:28
+
+from django.db import migrations, models
+
+# Separate state and database changes in migration to avoid downtime.
+# For next release will be prepared migration to remove the field from database.
+
+# Database fields in this migration are changed to null to allow adding new rows.
+# To reverse that operation, we will set all values for `SiteSettings` regarding the
+# first channel in alphabetical order.
+
+
+def update_order_settings_values_to_match_first_channel(apps, schema_editor):
+    SiteSettings = apps.get_model("site", "SiteSettings")
+    Channel = apps.get_model("channel", "Channel")
+
+    channel = Channel.objects.filter(is_active=True).order_by("slug").first()
+
+    SiteSettings.objects.update(
+        automatically_confirm_all_new_orders=(
+            channel.automatically_confirm_all_new_orders
+        ),
+        automatically_fulfill_non_shippable_gift_card=(
+            channel.automatically_fulfill_non_shippable_gift_card
+        ),
+    )
```

Comment:
I don't get why we need that as those fields (`automatically_confirm_all_new_orders` and `automatically_fulfill_non_shippable_gift_card`) are removed in this migration.

Reply:
That function is provided to enable unapplying the migration (it's used as reverse code in line 51)

Reply:
Thanks, I missed it!